contracts/scripts/util/arbitrum_gas_util.ts

```diff
@@ -0,0 +1,291 @@
+const fs = require('fs');
+/*
+ Web3 needs to be used over Ethers since web3 tx receipts
+ for Arbitrum correctly store the feeStats object.
+ Ethers' tx receipts aren't updated to store that yet.
+ */
+const Web3 = require('web3');
+require('dotenv').config();
```

Comment:
Should be able to use imports here.
```suggestion
import fs from 'fs';
/*
 Web3 needs to be used over Ethers since web3 tx receipts
 for Arbitrum correctly store the feeStats object.
 Ethers' tx receipts aren't updated to store that yet.
 */
import Web3 from 'web3';
import dotenv from 'dotenv';

dotenv.config();
```